pkg/app/piped/trigger/deployment.go

```diff
@@ -44,6 +44,7 @@ func (t *Trigger) triggerDeployment(
 	n, err := t.getNotification(app.GitPath)
 	if err != nil {
 		t.logger.Error("failed to get the list of mentions", zap.Error(err))
+		t.reportDeploymentFailed(app, fmt.Sprintf("failed to get the list of mentions %v", err), commit)
```

Comment:
Not just when missing the .pipe.yaml file, but also when it failures triggering, it's nice to report it. Don't you think so?

Reply:
Yes, I do 👍

Reply:
I mean:

```go
defer func() {
  if err != nil {
    r.reportDeploymentFailed(app, "", commit)
}()